docs/src/weights.md

```diff
@@ -41,6 +41,16 @@ w = ProbabilityWeights([0.2, 0.1, 0.3])
 w = pweights([0.2, 0.1, 0.3])
 ```
 
+### `ExponentialWeights`
+
+Exponential weights are a common form of temporal weights which assign exponentially greater
+weight to past observations.
```

Comment:
Exponential weights are essentially an easier-to-implement moving window calculation, so shouldn't this be greater weight to recent observations?

Reply:
Yes, this should probably be "Exponential weights are a common form of temporal weights which assigns exponentially decreasing weights to past observations"? On a similar note, does having the order of the values increasing rather than decreasing by default make sense (I typically have more recent observations at the end of an array or dataframe)?

Reply:
> I typically have more recent observations at the end of an array or dataframe

I'm guessing that's the more common case, so I think increasing weights is the right default.